spec/words_counted_spec.rb

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+require_relative "spec_helper"
+
+describe WordsCounted do
+  describe ".from_file" do
+    it "opens and reads a text file" do
+      counter = WordsCounted.from_file('spec/support/the_hart_and_the_hunter.txt')
+      expect(counter.token_count).to eq(139)
+    end
+
+    it "opens and reads a text file with options" do
+      counter = WordsCounted.from_file('spec/support/the_hart_and_the_hunter.txt', exclude: 'hunter')
```

Comment:
Line is too long. [101/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.